### DISCUSSION TITLE
Disabling "Sunday" when firstDayOfWeek is Monday

### DISCUSSION THREAD

**Discussion by alexbusuioc** [gpbl/react-day-picker#1143] [Support] [answered]
This is very simple to reproduce. Just set the DayPicker to start with Monday as firstDayOfWeek,
and try disabling "Sunday" (index 7)

<DayPicker
    firstDayOfWeek={1}
    disabledDays={[ { before: new Date()} , { daysOfWeek: [6,7]} ]}
 />

In the example above, only "Saturday" will be disabled

**Comment by alexbusuioc** [comment, marked as answer]
oh, silly me. Sunday has the index "0", not "7"

**Comment by CezarCobuz** [comment]
It can happen to anyone from EU working with US code  ^_^
